### PR TITLE
Deafening Zombie Military Pilots

### PR DESCRIPTION
**Pull request by DukePaulAtreid3s**
#### Summary
None 

#### Purpose of change

The Military Flight Helmet, when inactive and equipped by a player, totally deafens them. This PR fixes the oversight that Zombie Military Pilots, who have a 100% chance of wearing this item, are not actually deafened.

#### Describe the solution
I removed the “HEARS” tag from Zombie Military Pilots

#### Describe alternatives you've considered
Not doing this, or giving them bey

#### Testing
Shouted outside its sight range, it didn’t hear me.

#### Additional context

Look ma, I did a thing!

### CODE REVIEW

File: data/json/monsters/zed_soldiers.json [hunk 1]
```diff
@@ -332,7 +332,6 @@
     "burn_into": "mon_zombie_scorched",
     "flags": [
       "SEES",
```
Comment (GuardianDll) [inline data/json/monsters/zed_soldiers.json:334]:
```suggestion
    "flags": [ "SEES", "WARM", "GRABS", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "NOHEAD", "FILTHY" ],
```
did you not lint your changes?

File: data/json/monsters/zed_soldiers.json [hunk 2]
```diff
@@ -302,7 +302,7 @@
     "id": "mon_zombie_military_pilot",
     "type": "MONSTER",
     "name": { "str": "zombie military pilot" },
-    "description": "Once a pilot, it is dressed in basic military fatigues, aviator sunglasses, and a sturdy-looking helmet.",
+    "description": "Once a pilot, it is wearing a flightsuit and a sturdy-looking flight helmet. The head twithces and adjusts constantly, following any and every source of movement.",
```
Comment (TheShadowFerret) [inline data/json/monsters/zed_soldiers.json:305]:
```suggestion
    "description": "Once a pilot, it is wearing a flightsuit and a sturdy-looking flight helmet. The head twitches and adjusts constantly, following any and every source of movement.",
```

**Comment by Maleclypse** [comment]
I’ve spoken with other members of the team about how to fix that tests aren’t running on this and the only suggestion I’ve received is for you to close this and reopen it. I’m going to try doing that on my side but if that doesn’t work you’ll want to reopen it as a new PR.

**Comment by Maleclypse** [comment]
Tests appear to have scheduled so this appears to be working.

**Comment by DukePaulAtreid3s** [comment]
I am @DukePaulAtreides, and I approve this message.

**Pull request merged**